pendulum >= 1.5 seems to break plotille

```
    def convert(self, val):
        for t, f in reversed(self.converters.items()):
>           if isinstance(val, t):
E           TypeError: isinstance() arg 2 must be a type or tuple of types
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -25,6 +25,7 @@ addons:
 install:
   - pip install pipenv
   - pipenv install --dev
+  - pip install pytest-flake8 # in python 2.7 need to reinstall...
 
 script: PYTHONPATH=$PWD pipenv run pytest
 

--- a/setup.py
+++ b/setup.py
@@ -16,7 +16,7 @@ setup(
     packages=find_packages(),
     install_requires=[
         'six',
-        'pendulum',
+        'pendulum<1.5',
     ],
     author='Tammo Ippen',
     author_email='tammo.ippen@posteo.de',

--- a/setup.py
+++ b/setup.py
@@ -8,7 +8,7 @@ try:
 except IOError:
     long_description = open('README.md').read()
 
-version = '3.3'
+version = '3.4'
 
 setup(
     name='plotille',

